backend/src/south/south-mqtt/south-mqtt.ts

```diff
@@ -148,29 +147,38 @@ export default class SouthMQTT extends SouthConnector<SouthMQTTSettings, SouthMQ
 
       switch (associatedItem.settings.valueType) {
         case 'number':
-          await this.addValues([
-            {
-              pointId: associatedItem.name,
-              timestamp: messageTimestamp,
-              data: {
-                value: message.toString()
+          await this.addContent({
+            type: 'time-values',
+            content: [
+              {
+                pointId: associatedItem.name,
+                timestamp: messageTimestamp,
+                data: {
+                  value: message.toString()
+                }
               }
-            }
-          ]);
+            ]
+          });
           break;
         case 'string':
-          await this.addValues([
-            {
-              pointId: associatedItem.name,
-              timestamp: messageTimestamp,
-              data: {
-                value: message.toString()
+          await this.addContent({
+            type: 'time-values',
+            content: [
+              {
+                pointId: associatedItem.name,
+                timestamp: messageTimestamp,
+                data: {
+                  value: message.toString()
+                }
               }
-            }
-          ]);
+            ]
+          });
           break;
```

Comment:
Since these two cases are the same, they can be shortened as:
```ts
        case 'number':
        case 'string':
          await this.addContent({
            type: 'time-values',
            content: [
              {
                pointId: associatedItem.name,
                timestamp: messageTimestamp,
                data: {
                  value: message.toString()
                }
              }
            ]
          });
          break;
```